client/src/views/Header.tsx

```diff
@@ -80,41 +109,129 @@ function Header() {
           <NavLink to={routes.Home.path} className={classes.home}>
             <img src={Logo} alt="NEH logo placeholder" />
           </NavLink>
-          <NavLink className={classes.navLink} to={routes.AboutUs.path}>
+          <NavLink
+            className={classes.navLink}
+            to={routes.AboutUs.path}
+            activeStyle={{ fontWeight: 'bold' }}
+          >
             About Us
           </NavLink>
-          <NavLink className={classes.navLink} to={routes.HowItWorks.path}>
+          <NavLink
+            className={classes.navLink}
+            to={routes.HowItWorks.path}
+            activeStyle={{ fontWeight: 'bold' }}
+          >
             How It Works
           </NavLink>
-          <NavLink className={classes.navLink} to={routes.ContactUs.path}>
-            Contact Us
+          <NavLink
+            className={classes.navLink}
+            to={routes.Help.path}
+            activeStyle={{ fontWeight: 'bold' }}
+          >
+            FAQs
           </NavLink>
         </div>
         <div className={classes.userButtons}>
           {user ? (
             <>
-              <IconButton aria-label="user dropdown" onClick={handleClick} size="large">
+              <IconButton>
+                <img src={Bell} className={classes.bellIcon} />
+              </IconButton>
+              <IconButton
+                id="navigation-button"
+                aria-label="navigation dropdown"
+                aria-controls={open1 ? 'navigation-menu' : undefined}
+                aria-haspopup="true"
+                aria-expanded={open1 ? 'true' : undefined}
+                onClick={handleClick}
+                size="large"
+              >
+                <AppsIcon />
+              </IconButton>
+              <Menu
```

Comment:
i'd say each of these menus could be their own components
imo, they make this render pretty large and it'd be more consumable if they were broken out. what do you think?